apps/passport-server/test/generic-issuance/pipelines/pretix/pretixPipeline.spec.ts

```diff
@@ -510,7 +511,10 @@ describe("generic issuance - PretixPipeline", function () {
     expectToExist(adminUser);
 
     // Delete the manual tickets from the definition
-    const newPipelineDefinition = structuredClone(ethLatAmPipeline);
+    const latestPipeline = (await giService.getPipeline(
+      ethLatAmPipeline.id
+    )) as LemonadePipelineDefinition;
+    const newPipelineDefinition = structuredClone(latestPipeline);
```

Comment:
@robknight this may be relevant for future tests but basically this makes the test work with the clobbering-prevention feature